SocialCareCaseViewerApi.Tests/V1/Gateways/Database/GetPersonalRelationshipTypeByDescriptionTests.cs

```diff
@@ -0,0 +1,46 @@
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.V1.Gateways;
+using SocialCareCaseViewerApi.V1.Helpers;
+using Microsoft.EntityFrameworkCore;
+using System;
+
+namespace SocialCareCaseViewerApi.Tests.V1.Gateways.Database
+{
+    [TestFixture]
+    public class GetPersonalRelationshipTypeByDescriptionTests : DatabaseTests
+    {
+        private DatabaseGateway _databaseGateway;
+        private Mock<IProcessDataGateway> _mockProcessDataGateway = new Mock<IProcessDataGateway>();
+        private Mock<ISystemTime> _mockSystemTime = new Mock<ISystemTime>();
+
+        [SetUp]
+        public void Setup()
+        {
+            _databaseGateway = new DatabaseGateway(DatabaseContext, _mockProcessDataGateway.Object, _mockSystemTime.Object);
+            DatabaseContext.ChangeTracker.QueryTrackingBehavior = QueryTrackingBehavior.NoTracking;
+        }
+
+        [Test]
+        public void WhenNoMatchingDescriptionReturnsNull()
+        {
+            var response = _databaseGateway.GetPersonalRelationshipTypeByDescription("foobar");
+
+            response.Should().BeNull();
+        }
+
+        [Test]
+        [TestCase("parent")]
+        [TestCase("child")]
+        [TestCase("friend")]
+        public void WhenMatchingDescriptionReturnsPersonalRelationshipType(string description)
+        {
+            var response = _databaseGateway.GetPersonalRelationshipTypeByDescription(description);
```

Comment:
Just to confirm my understanding `parent`, `child` and `friend` are all relationship types we have stored in the database by default.

This is why this test is able to call `_databaseGateway.GetPersonalRelationshipTypeByDescription` before adding anything to the db first?

Reply:
Yup, that's exactly right - it's been added to the `schema.sql` which means we don't have to seed the database with personal relationship types.

https://github.com/LBHackney-IT/social-care-case-viewer-api/blob/5e5216b89ec53fa1e7b6a26119119e74a6faf083/database/schema.sql#L431-L499